### DISCUSSION TITLE
Tools configuration of RichTextToolBar

### DISCUSSION THREAD

**Discussion by simoco0781** [zalify/easy-email-editor#260] [Ideas]
Hi, is there any way to add/remove tools from the toolbar. 
Or may it be possible to just add "superscript" and "subscript" pending the replacement by Slate.js?

**Comment by m-Ryan** [comment]
Currently, there is no feature for adding/removing tools. If you are willing to provide a PR, we would be happy to accept it.

Perhaps it just needs to add more parameters, such as

```tsx
 toolbar?: {
    prefix?: React.ReactNode;
    list?:  Array<'bold'|'italic'...>;
    subfix?: React.ReactNode;
  };

```
